.astyleignore

```diff
@@ -23,3 +23,4 @@ targets
 components/802.15.4_RF
 components/wifi
 tools
+components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/tfm_impl
```

Comment:
👍 for the commit comment about this.

Reply:
One note: please revert file permissions changes `100644 → 100755` ?